homeassistant/components/iotty/switch.py

```diff
@@ -0,0 +1,122 @@
+"""Implement a iotty Light Switch Device."""
+
```

Comment:
Import annotations from __future__, in this file and in all other files
```suggestion
from __future__ import annotations

```

Reply:
This has not been fixed. Please add it to all files to avoid inconsistent behavior by the type checker.

Reply:
I didn't notice you asked to do that for all files, now it is done.